src/Core/System/Snippet/SnippetService.php

```diff
@@ -306,6 +306,15 @@ private function getSnippetsByLocale(SnippetFileCollection $snippetFileCollectio
     }
 
     /**
+     *  Collects snippet files for each given locale (ISO code).
```

Comment:
```suggestion
     *  Collects snippet files for each given locale
```
I'm not that much of a fan of "ISO code" and would like to avoid it as much es possible. ISO code is like "Yeah, there's a standard behind it, but I don't tell you which".

Very personal opinion, feel free to just resolve this.